File: src/main/java/com/uwetrottmann/trakt/v2/entities/Movie.java
package com.uwetrottmann.trakt.v2.entities;

import org.joda.time.DateTime;

import java.util.List;

public class Movie extends BaseEntity {

    public Integer year;
    public MovieIds ids;

    // extended info
    public String certification;
    public String tagline;
    public DateTime released;
    public Integer runtime;
    public String trailer;
    public String homepage;
    public String language;
    public List<String> genres;

}

File: src/main/kotlin/com/redissi/trakt/entities/Movie.kt
package com.redissi.trakt.entities

import com.squareup.moshi.JsonClass
import java.time.LocalDate
import java.time.OffsetDateTime

@JsonClass(generateAdapter = true)
class Movie(
    override val title: String? = null,
    override val overview: String? = null,
    override val rating: Double? = null,
    override val votes: Int? = null,
    override val updatedAt: OffsetDateTime? = null,
    override val availableTranslations: List<String>? = null,
    val year: Int? = null,
    val ids: MovieIds? = null,
    // extended info
    val certification: String? = null,
    val tagline: String? = null,
    /** Date in UTC time.  */
    val released: LocalDate? = null,
    val runtime: Int? = null,
    val trailer: String? = null,
    val homepage: String? = null,
    val language: String? = null,
    val genres: List<String>? = null
) : BaseEntity()